kpi/deployment_backends/kc_access/utils.py

```diff
@@ -30,7 +30,7 @@ def _trigger_kc_profile_creation(user):
     UserProfile if none exists already
     """
     url = settings.KOBOCAT_URL + '/api/v1/user'
-    token, _ = Token.objects.get_or_create(user=user)
+    token, _ = Token.objects.using('kobocat').get_or_create(user=user)
```

Comment:
A bit uncomfortable with this... Should we be allowing user creation(an onadata user) from KPI ?

Reply:
Ah, yeah, let me change that to a get request. It should already exist in Onadata by this point. Thanks for catching that